### DISCUSSION TITLE
Using CIBW to build external dependency wheels

### DISCUSSION THREAD

**Discussion by AbstractUmbra** [pypa/cibuildwheel#1343] [Q&A]
Hey there!

I'm currently undertaking a small project to help a community I'm in where I host a pip index for certain dependencies our project uses.
The reason being is that a lot of our users are inexperienced and aren't capable of building their wheels on Linux or Windows, and some wheels don't exist from the maintainers for 3.11 yet, but we experience similar issues on every minor version bump to Python.

I created [this](https://github.com/AbstractUmbra/abstractumbra.github.io/blob/main/.github/workflows/build_wheels.yaml) project to build wheels for a few OS' and architectures because I could not figure out how to do adapt this project to my needs. So I'm asking here if such a thing is possible?

TL;DR: can I use CIBW to build wheels for X, Y or Z modules/packages?
